core/transport/src/main/proto/grpc/table/layout/hive/hive.proto

```diff
@@ -6,7 +6,6 @@ option java_outer_classname = "HiveLayoutProto";
 
 package alluxio.grpc.table.layout;
 
-import "grpc/common.proto";
```

Comment:
why this change?

Reply:
Because when I tried to build the source, it complains some error like:
duplicated definition ....